pkg/oc/cli/admin/policy/remove_from_project.go

```diff
@@ -90,6 +95,16 @@ func (o *RemoveFromProjectOptions) Complete(f kcmdutil.Factory, cmd *cobra.Comma
 	o.Output = kcmdutil.GetFlagString(cmd, "output")
```

Comment:
That line should not be needed, right?

Reply:
We do end up using the value of this field, as this command mimics --dry-run behavior when an output value is given: https://github.com/openshift/origin/blob/master/pkg/oc/cli/admin/policy/remove_from_project.go#L163